Gifski/EditVideoViewController.swift

```diff
@@ -318,10 +342,43 @@ final class EditVideoViewController: NSViewController {
 			}
 		}()
 		let frameCount = duration * frameRateSlider.doubleValue
-		let dimensions = resizableDimensions.changed(dimensionsType: .pixels).currentDimensions.value
-		var fileSize = (Double(dimensions.width) * Double(dimensions.height) * frameCount) / 3
-		fileSize = fileSize * (qualitySlider.doubleValue + 1.5) / 2.5
-		estimatedSizeLabel.stringValue = "Estimated size: " + formatter.string(fromByteCount: Int64(fileSize))
+
+		estimatedSizeLabel.textColor = NSColor(red: 0.13, green: 0.13, blue: 0.13, alpha: 1.0)
+		estimatedSizeLabel.stringValue = "Starting Estimation..."
+		estimateWorkItem = DispatchWorkItem {
+			print("Total: \(frameCount)")
+
+			Gifski.setOnNewFrame { framesWritten in
+				DispatchQueue.main.sync {
+					self.estimatedSizeLabel.textColor = NSColor(red: 0.11, green: 0.37, blue: 0.13, alpha: 1.0)
```

Comment:
this is probably creating retain cycle as the `DispatchWorkItem` is retaining the parent by using `self` and the parent retains the work item by storing it as its property